### ISSUE TITLE
avoid deprecation warngin for createdb and fix postinstall script.

### ISSUE THREAD

**Issue by cornelinux** [NetKnights-GmbH/ubuntu#56]
Bummer.

The install of 3.8dev1 fails, I think since the post install script still uses `pi-manage createdb` which raises a Deprecation warning which seems to create a return value of 1.

Please change these to `create-table` and others accordingly.

it happens here:
https://github.com/NetKnights-GmbH/ubuntu/blob/master/debian_server/privacyidea-apache2.postinst#L38

and probably at other places.

~~~~
<SQLAlchemy engine=mysql+pymysql://pi:***@localhost/pi?charset=utf8>
/opt/privacyidea/bin/pi-manage:576: DeprecationWarning: This function will be removed in Version 4.0
  warnings.warn("This function will be removed in Version 4.0", DeprecationWarning)

             _                    _______  _______
   ___  ____(_)  _____ _______ __/  _/ _ \/ __/ _ |
  / _ \/ __/ / |/ / _ `/ __/ // // // // / _// __ |
 / .__/_/ /_/|___/\_,_/\__/\_, /___/____/___/_/ |_|
/_/                       /___/
                                          v3.8.dev1
    
The file 
	/etc/privacyidea/private.pem
already exist. We do not overwrite it!

             _                    _______  _______
   ___  ____(_)  _____ _______ __/  _/ _ \/ __/ _ |
  / _ \/ __/ / |/ / _ `/ __/ // // // // / _// __ |
 / .__/_/ /_/|___/\_,_/\__/\_, /___/____/___/_/ |_|
/_/                       /___/
                                          v3.8.dev1
    

             _                    _______  _______
   ___  ____(_)  _____ _______ __/  _/ _ \/ __/ _ |
  / _ \/ __/ / |/ / _ `/ __/ // // // // / _// __ |
 / .__/_/ /_/|___/\_,_/\__/\_, /___/____/___/_/ |_|
/_/                       /___/
                                          v3.8.dev1
    

             _                    _______  _______
   ___  ____(_)  _____ _______ __/  _/ _ \/ __/ _ |
  / _ \/ __/ / |/ / _ `/ __/ // // // // / _// __ |
 / .__/_/ /_/|___/\_,_/\__/\_, /___/____/___/_/ |_|
/_/                       /___/
                                          v3.8.dev1
    
dpkg: Fehler beim Bearbeiten des Paketes privacyidea-apache2 (--configure):
 »installiertes privacyidea-apache2-Skript des Paketes post-installation«-Unterprozess gab den Fehlerwert 1 zurück
Fehler traten auf beim Bearbeiten von:
 privacyidea-apache2
libdvd-pkg: Checking orig.tar integrity...
/usr/src/libdvd-pkg/libdvdcss_1.4.2.orig.tar.bz2: OK
libdvd-pkg: `apt-get check` failed, you may have broken packages. Aborting...
E: Sub-process /usr/bin/dpkg returned an error code (1)

~~~~

**Comment by cornelinux** [comment]
I issues a PR for the ubuntu packages.
@pablo-knight  This also needs to be done for redhat. Please ask @plettich for further details.

**Comment by plettich** [comment]
Fixed by  #57 and https://github.com/NetKnights-GmbH/centos7/pull/26
